subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -1391,6 +1286,73 @@ By default, Gradle caches changing modules for 24 hours. To change how long Grad
 
 For more details, take a look at the API documentation for api:org.gradle.api.artifacts.ResolutionStrategy[].
 
+[[sec:dependency_management_overview]]
+=== Dependency Management Best Practices
+
+While Gradle has strong opinions on dependency management, the tool gives you a choice between two options: follow recommended best practices or support any kind of pattern you can think of. This section outlines the Gradle project's recommended best practices for managing dependencies.
```

Comment:
Let's address this section with another PR. I just moved the existing content to later so that the flow starts to make sense again based on my proposed outline.